src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -21,8 +21,12 @@ internal static partial class Interop
 {
     internal static partial class OpenSsl
     {
+        // This cache size affects number of TLS Session Tickets each SslCtx will cache, not the number of SslCtx instances.
+        // Special value of 0 means unlimited, -1 means the implementation (OpenSSL) default, which is currently 20 * 1024.
         private const string TlsCacheSizeCtxName = "System.Net.Security.TlsCacheSize";
         private const string TlsCacheSizeEnvironmentVariable = "DOTNET_SYSTEM_NET_SECURITY_TLSCACHESIZE";
+        private const int DefaultTlsCacheSizeClient = 500; // since we keep only one TLS Session per hostname, 500 should be enough to cover most scenarios
+        private const int DefaultTlsCacheSizeServer = -1; // use implementation default
```

Comment:
if we have two different values I'm wondering if we should also split the variables...? The 500 for client seems like good default to me.

Reply:
I thought about that as well, but we either would have to
- rename the existing configuration knob (which could be argued to be a breaking change)
- introduce two more knobs to kontrol client/server separately
- introduce one new with a server/client suffix and reuse the existing one for the other role

and neither of the above is a clear winner so I wanted to not complicate things needlessly.